Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -700,6 +700,8 @@ namespace Babylon
             bgfx::setViewRect(0, 0, 0, w, h);
 #ifdef __APPLE__
             bgfx::frame();
+#elif __linux__
+            bgfx::discard();
```

Comment:
This is looking to be a bit concerning. Why is the code different for different platforms here?

Reply:
Still trying to understand what happens.

Reply:
Reverted change.